src/api/customer/customer-business.ts

```diff
@@ -11,6 +13,33 @@ const persistCustomer: CustomerCreationRequestHandler = async (req, res, next) =
     }
 }
 
+const persistFindCustomers: CustomersFindRequestHandler = async (req, res, next) => {
+    try {
+        const { customerToPK } = res.locals;
+        const users = await Customer.findByPk(customerToPK.uuid);
+        if(users != null){
```

Comment:
Não dá problema do modo que está.
Mas é uma boa prática em javascript usar os operadores `!==` e `===`, pois estes fazem checagem de valor e tipo.
Diferentemente de `!=` e `==`, que fazem checagem apenas de valor.

```suggestion
        if (users !== null) {
```